### DISCUSSION TITLE
Filament Fabricator: Layout "default" not found

### DISCUSSION THREAD

**Discussion by devhoussam1998** [Z3d0X/filament-fabricator#101] [Q&A]
Why I get **Filament Fabricator: Layout "default" not found** when I did not select any layout.

https://github.com/Z3d0X/filament-fabricator/assets/121255432/e324ea4e-7c2d-4374-bd02-f7b4100361d3

<img width="1385" alt="Screenshot 2023-10-14 at 01 00 04" src="https://github.com/Z3d0X/filament-fabricator/assets/121255432/6cc26451-c6a0-4afa-bc01-79416b6abd2b">
 

as you can see above why the validation message does not show up for layout?

**Comment by Z3d0X** [comment]
This was a bug, it is fixed in the new version just released: `v2.0.3` (ref: #103) please do a `composer update`



- Please note this would not fix the issue on existing records (but would prevent new records from being created without a layout selection)
